autometa/taxonomy/gtdb.py

```diff
@@ -57,9 +59,9 @@ def create_gtdb_db(reps_faa: str, dbdir: str) -> str:
         tar.close()
         logger.debug("Extraction done.")
         reps_faa = dbdir
-
+ 
     genome_protein_faa_filepaths = glob.glob(
-        os.path.join(reps_faa, "**", "*_protein.faa"), recursive=True
+        os.path.join(reps_faa, "**", "*_protein.faa.gz"), recursive=True
```

Comment:
Should this also find files that the user may have decompressed? Are these files still decompressed in earlier database versions? If so, we can provide backwards compatibility here

 ```suggestion
        os.path.join(reps_faa, "**", "*_protein.faa*"), recursive=True
        # To find *_protein.faa and *_protein.faa.gz files
```

Perhaps there is a literal glob character to optionally find `[.gz]` files instead of wildcard? 🤷 

```python
        os.path.join(reps_faa, "**", "*_protein.faa[.gz]"), recursive=True # I don't think this works?
```